compiler/semtempl.nim

```diff
@@ -261,6 +261,7 @@ proc semTemplSymbol(c: PContext, n: PNode, s: PSym; isField: bool): PNode =
   else:
     if isField and sfGenSym in s.flags: result = n
     else: result = newSymNode(s, n.info)
+    markUsed(c, n.info, s);
```

Comment:
It's not really "used" here IMO, but nimsuggest should be aware of it, some subcall of ``markUsed`` should be used, I think. Also, no semicolon here please.